tools/json_schema/json_schema-writers-inputs.adb

```diff
@@ -540,7 +540,11 @@ package body JSON_Schema.Writers.Inputs is
       Writers.Each_Property (Map, Schema, Write_When_Clause'Access);
 
       Put ("when others =>"); New_Line;
-      Put ("Success := False;"); New_Line;
+      Put ("loop"); New_Line;
```

Comment:
We should use `Reader.Skip_Value` here instead of because unknown value could contains another object inside and loop stops too early.

Reply:
Done